ManualTestsReport.md

```diff
@@ -0,0 +1,245 @@
+- Edit the private key in the client.
+
+| Step # | Action                                                     | Expected                                                      |
+|--------|------------------------------------------------------------|---------------------------------------------------------------|
+| 1      | Call the .exe file with all the parameters (see example 1) | Console message states: (see log4)   |
+| 2      | In destination path                                        | No new file has been created |
+
+log4:
+```
+Error connecting (check the connectionDataFile content)
+Invalid Private Key:
+```
+
+# Scenario 6: Cannot establish connection due to incorrect username
+
+**Preconditions**: 
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Edit the username in the connectionDataFile.
+
+| Step # | Action                                                     | Expected                                                      |
+|--------|------------------------------------------------------------|---------------------------------------------------------------|
+| 1      | Call the .exe file with all the parameters (see example 1) | Console message states: (see log5)   |
+| 2      | In destination path                                        | No new file has been created |
+
+log5:
+
+```
+Error connecting (check the connectionDataFile content)                                                                 
+Authentication failed using the private key: Access denied for 'publickey'. Authentication that can continue: password,keyboard-interactive,publickey  
+```
+
+# Scenario 7: Error case missing sourceFile
+
+**Preconditions**: 
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Remove the sourceFile from the path specified in the connectionDataFile.
+
+| Step # | Action                                                     | Expected                                                      |
+|--------|------------------------------------------------------------|---------------------------------------------------------------|
+| 1      | Call the .exe file with all the parameters (see example 1) | Console message states: (see log6)   |
+| 2      | In destination path                                        | No new file has been created |
+
+log6:
+
+```
+Error uploading the file: Local file does not exist or can not be read:  
+```
+
+# Scenario 8: Error case duplicate renameFilename in destination
+
+**Preconditions**: 
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Place a file with the same name as specified in renameFilename option.

```

Comment:
For me this precondition is not clear. When is this renameFilename specified?

Reply:
It is specified in the documentation of the library, which is missing in this branch because it is pending to be reviewed by Josep in his branch of documentation